betelgeuse.py

```diff
@@ -814,3 +814,134 @@ def test_run(
     pool.close()
     pool.join()
     TestRun.session.tx_commit()
+
+
+def create_xml_property(name, value):
+    """Create an XML property element and set its name and value attributes."""
+    element = ElementTree.Element('property')
+    element.set('name', name)
+    element.set('value', value)
+    return element
+
+
+@cli.command('xml-test-run')
+@click.option(
+    '--custom-fields',
+    help='Indicates to the importer which custom fields should be set. '
+    'Expected format is either id=value or JSON format {"id":"value"}. This '
+    'option can be specified multiple times.',
+    multiple=True,
+)
+@click.option(
+    '--dry-run',
+    help='Indicate to the importer to not make any change.',
+    is_flag=True,
+)
+@click.option(
+    '--lookup-method',
+    default='custom',
+    help='Indicates to the importer which lookup method to use. "id" for work '
+    'item id or "custom" for custom id (default).',
+    type=click.Choice([
+        'id',
+        'custom',
+    ])
+)
+@click.option(
+    '--not-finished',
+    help='Indicate to the importer to set the status to "In Progress" instead '
+    'of "Finished".',
+    is_flag=True,
+)
+@click.option(
+    '--not-include-skipped',
+    help='Indicate to the importer to no include skipped tests.',
+    is_flag=True,
+)
+@click.option(
+    '--test-run-id',
+    default='test-run-{0}'.format(time.time()),
+    help='Test Run ID to be created/updated.',
+)
+@click.option(
+    '--test-run-title',
+    help='Test Run title.',
+)
+@click.argument('junit-path', type=click.Path(exists=True, dir_okay=False))
```

Comment:
Can `arguments` use a `help` attribute? It would be great to explain what this argument is used for... same goes for `source-code-path`